packages/expo-ads-facebook/ios/EXAdsFacebook/EXAdSettingsManager.m

```diff
@@ -20,7 +20,6 @@ @implementation EXAdSettingsManager
 - (instancetype)init {
   if (self = [super init]) {
     _testDevices = [NSMutableArray new];
-    _urlPrefix = @"";
```

Comment:
annotate this property as `nonatomic, strong, nullable`

Reply:
https://github.com/expo/expo/pull/5932/commits/794633a69fac4234017d164d815b0fb66eb8f762 ✅